pkgs/development/python-modules/asciimatics/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+, pyfiglet
+, pillow
+, wcwidth
+, future
+, mock
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "asciimatics";
+  version = "1.10.0";
+
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    sha256 = "ccfc28a04ae39fa6f73bbf8a45cef99476df3fefafb451f67598453921c1d4ce";
+  };
+
+  buildInputs = [
+    setuptools_scm
+  ];
+
+  propagatedBuildInputs = [
+    pyfiglet
+    pillow
+    wcwidth
+    future
+  ];
+
+  checkInputs = [
+    mock
+    nose
+  ];
+
+  meta = with lib; {
+    description = "Asciimatics is a package to help people create full-screen text UIs (from interactive forms to ASCII animations) on any platform";
```

Comment:
We don't put the package name in the description to be terse, so these needs to be shortened.

Reply:
Should I remove the `checkInputs` then? Also I notice that `pytestrunner` is actually part of `setup_requires` so that should be in `buildInputs` right?